webv2/config/config_service.go

```diff
@@ -20,12 +20,40 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+	"path/filepath"
+	"sync"
 )
 
-var configFilePath string = "./webv2/config.json"
+var configFilePath string
+var configFilePathMutex sync.Mutex
 
+func getConfigFilePath() (string, error) {
+	dirPath := "spanner_migration_tool_output/config"
```

Comment:
nit: any reason not to call this just output ?

Reply:
This path is also used for storing the session state. [ref](https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/aa3d540e59aeaa7773aa254fed0f13270e91daed/webv2/session/local_session_store.go#L23).